driver/bpf/filler_helpers.h

```diff
@@ -947,6 +950,7 @@ static __always_inline bool bpf_in_ia32_syscall()
 #endif
 
 	return status & TS_COMPAT;
+#endif
```

Comment:
Please add `// __ppc64__` so it's obvious which #ifdef this #endif is matched with

Reply:
Thanks @nathan-b have added the // __ppc64__